metecho/api/models.py

```diff
@@ -489,6 +582,37 @@ def get_collaborator(self, gh_uid: str) -> Optional[Dict[str, object]]:
             return None
 
 
+class ProjectDependency(HashIdMixin, TimestampsMixin):
+    name = StringField()
+    url = models.URLField()
+    recommended = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = _("project dependency")
+        verbose_name_plural = _("project dependencies")
+        ordering = ("-recommended", "name")
+
+
+class GitHubOrganization(HashIdMixin, TimestampsMixin):
+    name = StringField()
+    login = StringField(unique=True, help_text="Organization's 'username' on GitHub")
+    avatar_url = models.URLField(_("Avatar URL"), blank=True)
```

Comment:
Should we add a note in the admin that this will be auto-populated after saving?

Reply:
Fixed in bbc5941ec643e043c127508eb49747a33c3d230c